### ISSUE TITLE
Collapsibles are closed by default, even though "active" class is applied.

### ISSUE THREAD

**Issue by jeroenqui** [Dogfalo/materialize#4375]
### Description
Collapsibles are closed by default when initialized, even though "active" class is applied.

### Repro Steps
See the demo page at: http://materializecss.com/collapsible.html#intialization
Section: Preselected Section
or
the codepen below

### Screenshots / Codepen
http://codepen.io/anon/pen/GWxYVP

**Comment by jfcere** [comment]
@acburst id like to bring your attention on this issue as it seems to come from https://github.com/Dogfalo/materialize/commit/b6d3188de5956fd5b20f6036abb59c096131a778#diff-31f7d6f4bbdc3b18ca4fef4a55f7dce4R78 ... 

If I remove the `toggleClass('active')` in the function `collapsibleOpen(object)` like this ...

```diff
// Open collapsible. object: .collapsible-header
function collapsibleOpen(object) {
- object.toggleClass('active');

  if (options.accordion || collapsible_type === "accordion" || collapsible_type === undefined) { // Handle Accordion
    accordionOpen(object);
  } else { // Handle Expandables
    expandableOpen(object);
  }

  execCallbacks(object);
}
```

and put it back in the `Add click handler to only direct collapsible header children` section like this ...
```diff
// Add click handler to only direct collapsible header children
$this.on('click.collapse', '> li > .collapsible-header', function(e) {
  var element = $(e.target);

  if (isChildrenOfPanelHeader(element)) {
    element = getPanelHeader(element);
  }

+ element.toggleClass('active');

  collapsibleOpen(element);
});
```

Everything start working back as normal ... what do you think?

**Comment by acburst** [comment]
Thanks for pointing this out! I'm sorry I didn't see your comment on the commit until now.

Fixed in b336331

**Comment by jeroenqui** [comment]
Maybe this is worth a new release? As it breaks the functionality of the menu on the materializecss.com site.
